pyproject.toml

```diff
@@ -24,16 +24,16 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.8.1"
 validator-collection = "^1.5.0"
-requests = "^2.28.0"
```

Comment:
Why is this coming out? Does the code no longer use it or was this accidental?

Reply:
it's a sceptre dependency.  we run `poetry install --extras sceptre` to get it.  Should it be explicitly added as a dependency?

Reply:
Ah, no, only if it's directly used by this package!